test/cases/adapters/postgresql/numeric_test.rb

```diff
@@ -0,0 +1,32 @@
+require "cases/helper_cockroachdb"
+
+# Load dependencies from ActiveRecord test suite
+require "support/schema_dumping_helper"
+
+module CockroachDB
+  class PostgresqlNumberTest < ActiveRecord::PostgreSQLTestCase
+    include SchemaDumpingHelper
+
+    class PostgresqlNumber < ActiveRecord::Base; end
+
+    setup do
+      @connection = ActiveRecord::Base.connection
+      @connection.create_table("postgresql_numbers", force: true) do |t|
+        t.decimal 'decimal_default'
+        t.decimal 'decimal_with_p', precision: 2
+        t.decimal 'decimal_with_p_and_s', precision: 2, scale: 2
+      end
+    end
+
+    teardown do
+      @connection.drop_table "postgresql_decimals", if_exists: true
+    end
+
+    def test_decimal_values
+      record = PostgresqlNumber.new(decimal_default: 111.222, decimal_with_p: 111.222, decimal_with_p_and_s: 111.222)
+      assert_equal record.decimal_default, 111.222
+      assert_equal record.decimal_with_p, 11
```

Comment:
Looks like this is failing because the ActiveRecord Types don't handle precision properly. Both `Type::DecimalWithoutScale` and `OID::Decimal`  allow any number of digits to the left of the decimal point for some reason. I can look into it more to see if using the `limit` parameter helps.

Reply:
I can just drop that test since it's not as important, it's really only the first assert that I'm actually worried about here.